File: Testing/Code for Testing/Early Basic Syntax Testing/macro/test_macro.js

function main() {
    let a = ((5) * 2);
    console.log(a);
    let b = ((3) + (7));
    console.log(b);
    let c = ((6) * (6));
    console.log(c);
    let d = ((((2) * 2)) + (((3) * (3))));
    console.log(d);
    return 0;
}

function __ul_sort(arr, n) { const sub = arr.slice(0, n).sort((a, b) => a < b ? -1 : a > b ? 1 : 0); for (let i = 0; i < n; i++) arr[i] = sub[i]; }

main();

File: Testing/Code for Testing/Early Basic Syntax Testing/macro/test_macro.py
from dataclasses import dataclass, field

def __ul_sort(arr, n): arr[:n] = sorted(arr[:n])

class __ul_Exception(Exception):
    def __init__(self, type_name, message):
        self.type_name = type_name
        self.message = message
        super().__init__(message)


def main():
    a = ((5) * 2)
    print(a)
    b = ((3) + (7))
    print(b)
    c = ((6) * (6))
    print(c)
    d = ((((2) * 2)) + (((3) * (3))))
    print(d)
    return 0


if __name__ == "__main__":
    main()
